service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/PertinentInformationObservationValueMapper.java

```diff
@@ -0,0 +1,191 @@
+        if (observationReferenceRangeComponent.hasText()) {
+            result += TEXT_PREFIX + observationReferenceRangeComponent.getText() + StringUtils.SPACE;
+
+            if (observationReferenceRangeComponent.hasLow()) {
+                SimpleQuantity low = observationReferenceRangeComponent.getLow();
+                if (low.hasValue()) {
+                    result += LOW_PREFIX + low.getValue() + StringUtils.SPACE;
+
+                    if (low.hasUnit()) {
+                        result += low.getUnit() + StringUtils.SPACE;
+                    }
+                }
+            }
+            if (observationReferenceRangeComponent.hasHigh()) {
+                SimpleQuantity high = observationReferenceRangeComponent.getHigh();
+                if (high.hasValue()) {
+                    result += HIGH_PREFIX + high.getValue() + StringUtils.SPACE;
+
+                    if (high.hasUnit()) {
+                        result += high.getUnit() + StringUtils.SPACE;
+                    }
+                }
+            }
+        }
+        return result;
+    }
+
+    private static String processCodeableConcept(CodeableConcept value) {
+        if (value.hasCoding() && !value.getCoding().isEmpty()) {
+            Coding coding = value.getCodingFirstRep();
+            if (coding.hasCode() && coding.hasDisplay()) {
+                return String.format(CODEABLE_CONCEPT_VALUE_TEMPLATE, coding.getCode(), coding.getDisplay());
+            }
+        }
+
+        return StringUtils.EMPTY;
+    }
+
+    private static String processBooleanType(BooleanType value) {
+        if (value.hasValue()) {
+            return String.format(BOOLEAN_VALUE_TEMPLATE, value.getValue());
+        }
+        return StringUtils.EMPTY;
+    }
+
+    private static String processRange(Range value) {
+        if (isLowValuePresent(value) && isHighValuePresent(value)) {
+            return String.format(RANGE_VALUE_TEMPLATE,
+                value.getLow().getValue(),
+                value.getLow().getUnit(),
+                value.getHigh().getValue(),
+                value.getHigh().getUnit());
+        }
+        return StringUtils.EMPTY;
+    }
+
+    private static String processRatio(Ratio value) {
+        if (isNumeratorPresent(value) && isDenominatorPresent(value)) {
+            Quantity numerator = value.getNumerator();
+            Quantity denominator = value.getDenominator();
+
+            return String.format(RATIO_VALUE_TEMPLATE,
+                numerator.getComparator().toCode(),
+                numerator.getValue(),
+                numerator.getUnit(),
+                denominator.getComparator().toCode(),
+                denominator.getValue(),
+                denominator.getUnit());
+        }
+
+        return StringUtils.EMPTY;
+    }
+
+    private static String processTimeType(TimeType value) {
+        if (value.hasValue()) {
+            return String.format(TIME_VALUE_TEMPLATE, value.getValue());
```

Comment:
Can we create a stub method in DateFormatUtil to convert BaseDateTimeType into String? It is fine to return getValue() for now, but this will ensure that we update to the correct format and time zone in NIAD-1082